File: BAEKJOON/Python/4000/4100/4101.py
"""
Title : 크냐
Link : https://www.acmicpc.net/problem/4101
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    while True:
        x, y = map(int, input().split())
        if x == y == 0:
            break
        if x > y:
            print("Yes")
        else:
            print("No")

File: BAEKJOON/Golang/4000/4100/4101.go
package main

import "fmt"

func main() {
	for {
		var x, y int
		fmt.Scanf("%d %d", &x, &y)
		if x == 0 && y == 0 {
			break
		}
		if x > y {
			fmt.Println("Yes")
		} else {
			fmt.Println("No")
		}
	}
}
